install/running_install.adoc

```diff
@@ -154,9 +154,11 @@ system can run out of files to open and the playbook will fail.
 
 To run the RPM-based installer:
 
-. Run the *_prequisites.yml_* playbook. This must be run only once before
-deploying a new cluster. Use the following command, specifying `-i` if your
-inventory file located somewhere other than *_/etc/ansible/hosts_*:
+. Run the *_prequisites.yml_* playbook. This playbook modifies the container
+runtimes and installs required software packages, if any. Unless you need to
```

Comment:
The playbook is used to to install required software packages first, then configure the container runtime accordingly. I personally think it would be better to adjust the order. And I prefer to use `configures the container runtimes` instead of `modifies the container runtimes`. 

Not a big deal anyway.